.ci_support/environment-docs.yml

```diff
@@ -3,7 +3,6 @@ channels:
 dependencies:
 - nbsphinx
 - dill
-- future
```

Comment:
This is fine and afaik unrelated to the imports down the line. Since we dropped python 2(?) `__future__` has been a built-in module.